fuse/data/ops/op_base.py

```diff
@@ -68,8 +89,28 @@ def reverse(self, sample_dict: NDict, key_to_reverse: str, key_to_follow: str, o
         :param key_to_follow: run the reverse according to the operation applied on this value
         :return: modified sample_dict
         """
-        raise NotImplemented
+        raise sample_dict
```

Comment:
I wonder what's better as a default - exception or not.

I understand the convenience of just inherit from OpReversibleOp to make it passthrough, but I wonder if it will make users mistake and think it auto-magically actually works and inverts the op.

Perhaps it would be better to throw an exception by default, but making it more detailed - like,
"... mark it safe for reverse by blah blah if there's nothing that should be done, or implement blah blah - see doc blah blah"